packages/store/tsconfig.json

```diff
@@ -1,13 +1,14 @@
 {
     "extends": "../../tsconfig.json",
-    "include": ["src", "test"],
+    "include": ["src"],
     "compilerOptions": {
         "baseUrl": "./src",
```

Comment:
```suggestion

```
Shall we remove this and the `paths`? Looks like we decided against using them. We don't need the config anymore

Reply:
Done